src/neo/Cryptography/Helper.cs

```diff
@@ -119,6 +121,63 @@ public static byte[] Sha256(this Span<byte> value)
             return Sha256((ReadOnlySpan<byte>)value);
         }
 
+        public static byte[] AES256Encrypt(this byte[] plainData, byte[] key, byte[] nonce, byte[] associatedData = null)
+        {
+            var keyLen = key is null ? 0 : key.Length;
+            var nonceLen = nonce is null ? 0 : nonce.Length;
+            if (keyLen != 32) throw new ArgumentException();
+            if (nonceLen != 12) throw new ArgumentException();
+            var msgLen = plainData is null ? 0 : plainData.Length;
+            var tagLen = 16;
+            var cipherBytes = new byte[msgLen];
+            var tag = new byte[tagLen];
+            using var cipher = new AesGcm(key);
+            cipher.Encrypt(nonce, plainData, cipherBytes, tag, associatedData);
+            var cipherWithTag = new byte[nonceLen + msgLen + tagLen];
+            Buffer.BlockCopy(nonce, 0, cipherWithTag, 0, nonceLen);
+            Buffer.BlockCopy(cipherBytes, 0, cipherWithTag, nonceLen, msgLen);
+            Buffer.BlockCopy(tag, 0, cipherWithTag, nonceLen + msgLen, tagLen);
+            return cipherWithTag;
+        }
+
+        public static byte[] AES256Decrypt(this byte[] encryptedData, byte[] key, byte[] associatedData = null)
+        {
+            if (key.Length != 32) throw new ArgumentException();
```

Comment:
Check `encryptedData` length?

Reply:
The key length,which is checked here.  It should be 32 bytes when we use  a AES gcm mode